exotic/api/elca.py

```diff
@@ -714,8 +714,10 @@ def loglike(pars):
         for n in range(nobs):
             self.lc_data[n]['errors'] = {}
             
-            # copy global parameters
-            self.lc_data[n]['priors'] = copy.deepcopy(self.parameters)
+            # set global parameters without overwriting everything
+            for gk in gfreekeys:
+                self.lc_data[n]['priors'][gk] = self.parameters[gk]
+                self.lc_data[n]['errors'][gk] = self.errors[gk]
 
```

Comment:
This is a positive change...

Reply:
Yes, unfortunately this was also overwriting the limb darkening coefficients which isn't ideal for multi-wavelength studies. Separating them out should preserve their original values and make the mosaic plot look.

Reply:
Yes, deep copy operations can be dangerous and costly!